app/lib/content_security_policy.rb

```diff
@@ -31,6 +31,10 @@ def sso_host
 
   private
 
+  def extra_data_hosts
+    ENV.fetch('EXTRA_DATA_HOSTS', '').split(',')
```

Comment:
I would support space-separated and comma-separated lists for consistency with `ALLOWED_PRIVATE_ADDRESSES` and `TRUSTED_PROXY_IP`.
```suggestion
    ENV.fetch('EXTRA_DATA_HOSTS', '').split(/(?:\s*,\s*|\s+)/)
```

Reply:
Thanks Claire.